ansible_base/jwt_consumer/common/auth.py

```diff
@@ -196,7 +196,7 @@ def authenticate(self, request):
 
             return user, None
         else:
-            return None, None
+            return None
```

Comment:
@Dostonbek1 This is the difference between DRF Authentication and Django Authentication. The DRF authentication returns a Tuple (user, token) while the Django Authentication returns a single value(user) . Is the context for JWTAuthentication DRF or Django?